test/extended/router/oc_copy.go

```diff
@@ -15,3 +15,13 @@ func IngressConditionStatus(ingress *routev1.RouteIngress, t routev1.RouteIngres
 	}
 	return corev1.ConditionUnknown, routev1.RouteIngressCondition{}
 }
+
+func PodConditionStatus(pod *corev1.Pod, t corev1.PodConditionType) corev1.ConditionStatus {
```

Comment:
Any reason to make this public?  It isn't a big deal, but we might as well make it private (lower-cased).